src/Overcrowdin/Overcrowdin.csproj

```diff
@@ -14,10 +14,11 @@
     <Description>.NET Core Global Tool providing a limited but useful Crowdin CLI. Built over the
 Crowdin.Api [Copyright (c) 2019 crowdin]</Description>
     <PackageLicenseUrl>https://opensource.org/licenses/MIT</PackageLicenseUrl>
```

Comment:
Should use the `<PackageLicenseExpression>MIT</PackageLicenseExpression>` element - `PackageLicenseUrl` is obsolete IIRC.

Reply:
Ah so that is how you get it to generate that part of the nuspec correctly.

Reply:
I used the license expression in the nuspec but Nuget.org still complained about my project not having a license when I uploaded.

Reply:
It's possible that there is a difference between using `nuget pack` and the .nuspec file and letting msbuild or dotnet create the nuget package from the .csproj file.